src/Sylius/Component/Association/Model/Associatable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+     * This file is part of the Sylius package.
+     *
+     * (c) Paweł Jędrzejewski
+     *
+     * For the full copyright and license information, please view the LICENSE
+     * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Association\Model;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface Associatable
```

Comment:
*Interface

I would like to have better named interfaces in whole Sylius too, but please do not change conventions we have now. It's better to follow a single not perfect convention than mixing better/worse conventions. We might change it in the future versions.